docs/tutorials/create-a-pipeline-that-blurs-faces-in-an-image.md

```diff
@@ -42,117 +42,189 @@ documentation to start them.
+    FastAPI documentation of the running [Core engine](../reference/core-engine.md)
+    and use the `/pipelines` endpoint to post the
+    [Pipeline](../reference/core-concepts/pipeline.md) by clicking on the
+    `Try it out` button
+    
+    Simply copy the content of the `face-blur-pipeline.json` file and paste it in
+    the `body` field of the `/pipelines` endpoint and click on the `Execute` button.
+    
+    ![Post pipeline](../assets/screenshots/post-pipeline.png)
+    
+    You should receive a `200` response with the
+    [Pipeline](../reference/core-concepts/pipeline.md) you just posted.
+
+=== "Using the Core engine interface"
+
+    Go to the Webapp on your browser to the  "/createPipeline" page of the [Core engine](../reference/core-engine.md).
+    From here you can chose from all available services to add to the new pipeline. First select the Face detection 
+    service. You can use the use the searchbar and filters on the side to help you find it.
+
+    ![Select Face detection](../assets/screenshots/pipeline-creation-select-face-detection.png)
+    
+    Click the "Add" button of the service. This will a dd a new new node in the flow chart. 
+
+    ![Flow Chart service](../assets/screenshots/pipeline-creation-flow-chart-add-service.png)
+
+    Do the same with the Image Blur service. You should now have 4 nodes in the flow chart. You can click on the nodes to drag and 
+    move them around to rearrange their disposition.
+    
+
+    There are 3 types of nodes:
+
+    - Entry node. It defines the input files the pipeline needs, and their types. Each input file has a handle you can connect 
+    to a service to pass on the data as input for the service.
+
+    - Exit node. It is the last node of the pipeline. Only one service (one handle) can be connected to it. The service connected to it 
+    defines what the output of the pipeline as whole will be.
+
+    - Service node. Each one represents a distinct service. For each data input the service needs, there is a handle you can connect to a data source, either the output of another service or
+    the input files of the pipeline defined by the entry node.
+    
+
+    Click the "Add input file" button on entry node, select a name for your input, and select the types image/jpeg and image/png.
+    
+    The new input can now be connected to the services. Each takes an image, in this case it will be the same image. Click on the handle 
+    next to the input on the entry Node and drag your mouse to the handle next to the service's input called "image". Do so for both services.
+
+
+    ![Connect input to services](../assets/screenshots/pipeline-creation-connect-input-to-services.png)
+    
+    There are two steps to the pipeline. The first one detects faces in a given image and the second blurs them. The output of the 
+    first service can be given to the second for the blurring process. To do so link the output of Face detection to the input of 
```

Comment:
"Face detection" should be "Face Detection" just like Image Blur below, with caps at the start. Do so on the whole tutorial